custom_model_runner/datarobot_drum/resource/components/Python/r_fit/fit.R

```diff
@@ -46,3 +46,17 @@ outer_fit <- function(X, y, output_dir, class_order=NULL, row_weights=NULL) {
             stop(sprintf("No Fit method provided."))
         }
 }
+
+
+outer_fit_unsupervised <- function(X, output_dir, row_weights=NULL) {
+    if (!isFALSE(fit_hook)) {
```

Comment:
theres no isTRUE?

Reply:
did some testing - looks like `!isFALSE ` and `isTRUE` don't behave the same, because R is the worst